.bumpversion.cfg

```diff
@@ -0,0 +1,20 @@
+[bumpversion]
+current_version = 2.18.1
+parse = (?P<major>\d+)
+        \.(?P<minor>\d+)
+        \.(?P<patch>\d+)
+        (\.(?P<release>[a-z]+)\.(?P<build>\d+))?
+
+serialize = 
+	{major}.{minor}.{patch}.{release}.{build}
+	{major}.{minor}.{patch}
+
+[bumpversion:part:release]
```

Comment:
we can move all the references to `build` and `release` as we should never have beta versions of this library.